src/Symfony/Component/Encryption/Tests/Phpseclib/PhpseclibEncryptionTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Encryption\Tests\Encryption\Phpseclib;
+
+use phpseclib\Crypt\AES;
+use Symfony\Component\Encryption\EncryptionInterface;
+use Symfony\Component\Encryption\KeyInterface;
+use Symfony\Component\Encryption\Phpseclib\PhpseclibEncryption;
+use Symfony\Component\Encryption\Phpseclib\PhpseclibKey;
+use Symfony\Component\Encryption\Tests\AbstractEncryptionTest;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class PhpseclibEncryptionTest extends AbstractEncryptionTest
```

Comment:
I would make the tests final, but just a personal preference

Reply:
we don't really care about putting `final` on tests: that does have any impact on maintenance